### DISCUSSION TITLE
How to retrieve patient fhir resource from questionnaire response bundle?

### DISCUSSION THREAD

**Discussion by mansikalraa** [google/android-fhir#2160] [Q&A] [answered]
When we extract data from the questionnaire response using Resource Mapper, we get bundle in return. [[ref.](https://github.com/google/android-fhir/wiki/SDCL%3A-Use-ResourceMapper)]
Could you please provide an example on how to retrieve the patient fhir response from the questionnaire response bundle got from the Resource mapper? So that we can use that patient response to sync using FHIR Engine.

@jingtang10

**Comment by fredhersch** [comment, marked as answer]
Hi @mansikalraa - have you looked at the example in the codelab on [developers.google.com/open-health-stack/learn ](https://developers.google.com/open-health-stack/codelabs/data-capture#5)

cc: @jingtang10

**Comment by mansikalraa** [comment]
Thank you. @fredhersch
